app/js/script.js

```diff
@@ -211,9 +198,40 @@ const showCaseSuccess = () => {
   caseError.textContent = '';
 };
 
+// Store and retrieve user information
+
+// Store user info
+const storeUserInfo = () => {
+  const theUser = {};
+  const userName = document.querySelector('#name').value;
+  const userEmail = document.querySelector('#email').value;
+  const userMessage = document.querySelector('#message').value;
+
+  theUser.name = userName;
+  theUser.email = userEmail;
+  theUser.message = userMessage;
+
+  // Saving to local storage
+  localStorage.setItem('theUser', JSON.stringify(theUser));
+};
+
+// Get user info
+const userInfo = JSON.parse(localStorage.getItem('theUser'));
+
+const retrieveUserInfo = () => {
+  if (userInfo !== 'null') {
+    document.querySelector('#name').value = userInfo.name;
+    document.querySelector('#email').value = userInfo.email;
+    document.querySelector('#message').value = userInfo.message;
+  }
+};
+
+retrieveUserInfo();
```

Comment:
[OPTIONAL] I think rather than calling it here we can use a reload event like `window.onload` and call the retrieve function then.

Reply:
Thanks for the suggestion, @PinkMoon25. I'll try that.